java/org/contikios/cooja/radiomediums/UDGM.java

```diff
@@ -188,8 +188,6 @@ protected RadioConnection createConnections(Radio sender) {
            it will be activated later when the radio will be
            turned on and switched to the right channel. This behavior
            is consistent with the case when receiver is turned off. */
```

Comment:
I know there is a lot of duplicated code in the radio mediums, should other radio mediums also have a similar change made?

Reply:
I made analogous changes to the other RadioMediums.